src/ducks/transactions/__snapshots__/TransactionModal.spec.jsx.snap

```diff
@@ -4,6 +4,7 @@ exports[`transaction modal should render correctly 1`] = `
 "<div class=\\"styles__c-modal-container___31zmi\\">
   <div class=\\"styles__c-overlay___31iWd\\">
     <div class=\\"styles__c-modal-wrapper___8PyE9\\">
+      <div></div>
```

Comment:
do you know why this change? I can't see anything related?

Reply:
https://github.com/cozy/cozy-ui/blob/master/react/Modal/index.jsx#L111